### PR TITLE
[479] CloudWatchEvent renamed to EventBridgeEvent

### LINKED ISSUE #479: CloudWatchEvent should be renamed
I think because the [documentation](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-what-is.html) suggests using EventBridge over CloudWatchEvent, I thought that it is better to rename CloudWatchEvent to something that includes EventBridge maybe?

### PR DESCRIPTION
**Pull request by mateusmarquezini**
*Issue #479*

*Description of changes:*

Hi there! This should fix #479. Please, let me know if you meant something else! 

Besides having renamed from CloudWatchEvent to EventBridgeEvent on some places of the code base, I also updated some links (despite they're still redirecting correctly)

Looking forward to your feedback! Thanks! :)

### CODE REVIEW

File: events/cloudwatch_events.go [hunk 1]
```diff
@@ -5,9 +5,9 @@ import (
 	"time"
 )
 
-// CloudWatchEvent is the outer structure of an event sent via CloudWatch Events.
-// For examples of events that come via CloudWatch Events, see https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/EventTypes.html
-type CloudWatchEvent struct {
```
Comment (bmoffatt) [inline events/cloudwatch_events.go:10]: Renaming the type is a breaking change. Adding a type alias `type EventBridgeEvent = CloudWatchEvent` would be required to not break existing consumers

File: events/cloudwatch_logs.go [hunk 2]
```diff
@@ -7,19 +7,19 @@ import (
 	"encoding/json"
 )
 
-// CloudwatchLogsEvent represents raw data from a cloudwatch logs event
```
Comment (bmoffatt) [inline events/cloudwatch_logs.go:10]: CloudWatch Events (now named Event Bridge) is distinct from CloudWatch Logs (still called CloudWatch!)

A find/replace of CloudWatch -> EventBridge isn't the way to go about this documentation refresh

File: events/codepipeline_cloudwatch.go [hunk 3]
```diff
@@ -42,8 +42,8 @@ const (
 )
 
 // CodePipelineEvent is documented at:
-// https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/EventTypes.html#codepipeline_event_type
-type CodePipelineCloudWatchEvent struct {
+// https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-service-event.html#codepipeline_event_type
+type CodePipelineEventBridgeEvent struct {
```
Comment (bmoffatt) [inline events/codepipeline_cloudwatch.go:46]: requires a type alias to not break existing consumers

File: events/codepipeline_cloudwatch.go [hunk 4]
```diff
@@ -42,8 +42,8 @@ const (
 )
 
 // CodePipelineEvent is documented at:
-// https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/EventTypes.html#codepipeline_event_type
-type CodePipelineCloudWatchEvent struct {
+// https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-service-event.html#codepipeline_event_type
```
Comment (bmoffatt) [inline events/codepipeline_cloudwatch.go:45]: The content is different than what it was when it was added in the first place. May as well remove the link if there's not an equivalent anymore

**Review by bmoffatt — CHANGES_REQUESTED** [review]

**Comment by mateusmarquezini** [comment]
Thank you for taking the time to suggest improvements @bmoffatt! I just made some changes as you suggested. Now it is as it should I guess. Let me know if it still needs more changes.

File: events/cloudwatch_events.go [hunk 5]
```diff
@@ -18,3 +17,5 @@ type CloudWatchEvent struct {
 	Resources  []string        `json:"resources"`
 	Detail     json.RawMessage `json:"detail"`
 }
+
+type EventBridge = CloudWatchEvent
```
Comment (bmoffatt) [inline events/cloudwatch_events.go:21]: `EventBridgeEvent`

File: events/README_CloudWatch_Events.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 
 # Sample Function
 
-The following is a Lambda function that receives Amazon CloudWatch event record data as input and writes event detail to Lambda's CloudWatch Logs. Note that by default anything written to Console will be logged as CloudWatch Logs events.
+The following is a Lambda function that receives Amazon EventBridge event record data as input and writes event detail to Lambda's CloudWatch Logs. Note that by default anything written to Console will be logged as CloudWatch Logs events.
```
Comment (bmoffatt) [inline events/README_CloudWatch_Events.md:5]: should rename this file too, README_EventBridge_Events.md

**Review by bmoffatt — CHANGES_REQUESTED** [review]

File: events/cloudwatch_logs_test.go [hunk 7]
```diff
@@ -10,10 +10,10 @@ import (
...
 func TestCloudwatchLogs(t *testing.T) {
 	for _, test := range []struct {
-		name                      string
-		eventJSON                 string
-		expectError               bool
-		expectCloudwatchEventData CloudwatchLogsEvent
+		name                  string
+		eventJSON             string
+		expectError           bool
+		expectEventBridgeData CloudwatchLogsEvent
```
Comment (bmoffatt) [inline events/cloudwatch_logs_test.go:16]: revert the changes to this test file, it's for CloudWatch Logs, not EventBridge

**Review by bmoffatt — CHANGES_REQUESTED** [review]

**Review by bmoffatt — APPROVED** [review]

**Pull request merged**